rebasehelper/application.py

```diff
@@ -706,6 +709,8 @@ def run(self):
                         elif self.conf.buildtool == 'copr':
                             self.print_copr_logs()
                         return 0
+                    if not build:
+                        return 1
```

Comment:
This should not be needed, since if building fails, the RebaseHelperError should be raised instead.